setup.py

```diff
@@ -169,7 +169,7 @@ def CythonExtension(module_name, **kw):
     ],
     packages=find_packages(where="src"),
     package_dir={"": PACKAGE_DIR},
-    install_requires=["numba==0.48.0", "numpy", "scitrack", "tqdm", "tinydb"],
```

Comment:
OK, now that I see the reason for the errors is a refactor in `numba`, we **do** need to define a minimum version here, I think `numba>0.48` will do it.